src/app/styles/css/mixins/variables.css

```diff
@@ -7,6 +7,7 @@
   --fontSizeSmall: 12px;
   --fontSizeDefault: 14px;
   --fontSizeBaseText: 16px;
+  --iconSizeTiny: 16px;
```

Comment:
@caiosba I'm not sure how necessary this is versus, just using the variable above `--fontSizeBaseText`
Considering icons are sized using font size. 🤷 not a huge deal, but a bit redundant

Reply:
@brianfleming yeah, thanks for clarifying, I had this same question when I was working on it... but since I saw both `fontSize*` and `iconSize*` there, I got confused...